getgist/github.py

```diff
@@ -42,14 +42,20 @@ def __init__(self, user, file_path, assume_yes=False):
         Save basic variables to all methods, instantiate GetGistrequests and
         calls the OAuth method.
         :param user: (str) GitHub username
-        :param file_path: (str) file_path to be saved (locally), created or
-        updated (remotelly)
+        :param file_path: (str or None) file_path to be saved (locally), created or
+        updated (remotelly) 
```

Comment:
There's a trailing unneeded space in this line.
```suggestion
        updated (remotelly)
```